bot/exts/holidays/valentines/be_my_valentine.py

```diff
@@ -41,21 +41,39 @@ async def lovefest_role(self, ctx: commands.Context) -> None:
         """
         raise MovedCommandError(MOVED_COMMAND)
 
-    @commands.cooldown(1, 1800, commands.BucketType.user)
+    # @commands.cooldown(1, 1800, commands.BucketType.user)
```

Comment:
I assume this was done while testing, but should be reverted now. To keep git history clean, avoid committing temporary changes.